testsuite/rabbitmq/classes.py

```diff
@@ -0,0 +1,88 @@
+import dataclasses
```

Comment:
move it to testsuite.databases

Reply:
That's what i did initially, but ain't it a bit misleading/not optimally structured?
I mean, RabbitMQ for sure is not a database

Should the docs be moved into databases section as well?

Reply:
perhaps it should be renamed someday to services. but let's leave it for now.